### PR TITLE
libs: added  cbor serialization

### LINKED ISSUE #82: add cbor implementation
#### Description

Is necessary serialization to message before send to the receiver board for this will be used tinycbor.

1) will be add an encode funtion  for  all value sensor.
2) will be add a decode function for decode the data recived since the emisor board 

### Useful links

Documentation of RIOT for tinycbor: https://doc.riot-os.org/group__pkg__tinycbor.html

### PR DESCRIPTION
**Pull request by xkevin190**
### Contribution description

Was added  tinycbor  and added the serialization functions also was add the libs folder.

### Issues/PRs references
this PR  fixes #82

### CODE REVIEW

File: firmware/libs/serialization/include/serialization.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    int16_t temperature;   /*!< temperature value */
+    int mositure;          /*!< moisture value */
+} sensor_data;
+
+
+/**
+ * @brief  This function decode message to arrive since udp
+ * @param[in]   buffer  data codified
+ * @param[out]  data    output of the decodified data
+ * @param[in]   length  length of the data
```
Comment (luisan00) [inline firmware/libs/serialization/include/serialization.h:51]:
```suggestion
 * @param [in]   buffer  data codified
 * @param [out]  data    output of the decodified data
 * @param [in]   length  length of the data
```

**Review by luisan00 — CHANGES_REQUESTED** [review]
good job man!, only some minor things

**Comment by luisan00** [comment]
no issue linked?

**Comment by xkevin190** [comment]
yes i linked the issue #83  @luisan00

**Comment by luisan00** [comment]
> yes i linked the issue #83 @luisan00

not, there is no link

**Comment by xkevin190** [comment]
ohh little mistake.

**Comment by luisan00** [comment]
> ohh little mistake.

I think it can be fixed , change it

**Comment by luisan00** [comment]
> ohh little mistake.

Same here:

Captura de pantalla de 2022-01-13 21-18-52

**Comment by luisan00** [comment]
> > ohh little mistake.
> 
> I think it can be fixed , change it

change ir

**Comment by luisan00** [comment]
Please do a squash before to approve

**Pull request merged**